src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,153 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //----------------------------------------------------------------------------
+  // Container
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
```

Comment:
just want to confirm what the overflow behavior should be, esp at smaller width. will scrolling be a table variant or would it be part of the core table behavior? other than that and @joshblack's open questions, looks good to me

Reply:
For small screen real estate - I have seen an approach reducing the column set by the earliest/biggest Carbon consumer, and thus I think that's what we'd recommend.